syntax = "proto3";

message HandShake {
    string hash = 1;
}
